spec.html

```diff
@@ -7692,7 +7692,7 @@ <h1>Static Semantics: Contains</h1>
         1. If _symbol_ is |ClassBody|, return *true*.
         1. If _symbol_ is |ClassHeritage|, then
           1. If |ClassHeritage| is present, return *true*; otherwise return *false*.
-        1. Let _inHeritage_ be |ClassHeritage| Contains _symbol_.
+        1. If |ClassHeritage| is present, let _inHeritage_ be |ClassHeritage| Contains _symbol_; otherwise let _inHeritage_ be false.
```

Comment:
```suggestion
        1. If |ClassHeritage| is present, let _inHeritage_ be |ClassHeritage| Contains _symbol_; otherwise let _inHeritage_ be *false*.
```

Reply:
That said, it's probably nicer to replace both this line and the subsequent one with

```
1. If |ClassHeritage| is present, then
  1. If |ClassHeritage| Contains _symbol_, return *true*.
```

Reply:
It looks to be cleaner.